CDP4.COMET.HERMES/Components/Server/ServerAuthenticationComponent.razor

```diff
@@ -0,0 +1,101 @@
+﻿<!--------------------------------------------------------------------------------------------------------------------
+    // <copyright file="ServerAuthenticationComponent.razor" company="RHEA System S.A.">
```

Comment:
indentation issue on header. Note: On razor component, the <copyright> with filename is not needed